src/transformers/models/instructblip/modeling_instructblip.py

```diff
@@ -1038,6 +1038,7 @@ def forward(
         else:
             embeddings = query_embeds
 
+        embeddings = embeddings.to(self.layernorm.weight.dtype)
         embeddings = self.layernorm(embeddings)
         embeddings = self.dropout(embeddings)
         return embeddings
```

Comment:
SHouldn't this be put back to the original dtype? Or is the layernorm in the right dtype.